app/views/posts/show.html.erb

```diff
@@ -22,4 +22,21 @@
       </li>
     <% end %>
   </ul>
-</div>
\ No newline at end of file
+</div>
+
+<h3>Create a Comment</h3>
+
+<%= form_with(model: [@post, Comment.new], local: true, url: user_post_comments_path(current_user, @post)) do |form| %>
+  <div class="field">
+    <%= form.text_area :text, placeholder: 'Enter your comment here' %>
+  </div>
+  <div class="actions">
+    <%= form.submit 'Create Comment' %>
+  </div>
+<% end %>
+<%= form_with(model: [current_user, @post.likes.build], local: true, url: user_post_likes_path(current_user, @post)) do |form| %>
+  <div class="actions">
+    <%= form.hidden_field :user_id, value: current_user.id %>
+    <%= form.submit 'Like', class: 'like-button' %>
+  </div>
```

Comment:
- [ ] It would be nice if you can update your design to show name the username of who created the post and comment, please see the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) 👍🏿 


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![blog_single_post](https://github.com/shm04/blogApp/assets/69638013/e644e713-bed6-413a-b6dd-0e55ce345116)  |  ![Screenshot_208](https://github.com/shm04/blogApp/assets/69638013/88884bc3-66fa-4cb3-b708-3e21d5629302)

Reply:
This change has not been done well.

- [ ] It would be nice if you can update your design to show name the username of who created the post please see the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) 👍🏿